scripts/packaging/update_all_worlds.py

```diff
@@ -43,4 +43,4 @@
 
 for i in range(len(worlds)):
     print('%d/%d: %s' % (i + 1, len(worlds), worlds[i]))
-    call([webotsFullPath, worlds[i], '--minimize', '--batch', '--no-sandbox', '--mode=pause', '--update-world'])
+    call([webotsFullPath, worlds[i], '--minimize', '--batch', '--mode=pause', '--update-world'])
```

Comment:
Was the `--no-sandbox` command removed or didn't exist to begin with? Either way there's still a reference to it in webots_ros [here](https://github.com/cyberbotics/webots_ros/blob/d0e2131164b5b18940a4548d93d0fc414d26020c/scripts/webots_launcher.py#L46)

Reply:
it has been removed since it was a command related to the WebEngine debugging. So it should also be removed in webot_ros. I am creating a PR.